packages/cli/src/workflows/workflow.service.ts

```diff
@@ -43,6 +43,24 @@ export type WorkflowsGetSharedOptions =
 
 @Service()
 export class WorkflowService {
+	constructor(
+		private readonly logger: Logger,
+		private readonly externalHooks: ExternalHooks,
+		private readonly internalHooks: InternalHooks,
+		private readonly executionRepository: ExecutionRepository,
+		private readonly sharedWorkflowRepository: SharedWorkflowRepository,
+		private readonly workflowRepository: WorkflowRepository,
+		private readonly workflowTagMappingRepository: WorkflowTagMappingRepository,
+		private readonly binaryDataService: BinaryDataService,
+		private readonly ownershipService: OwnershipService,
+		private readonly tagService: TagService,
+		private readonly workflowHistoryService: WorkflowHistoryService,
+		private readonly activeWorkflowRunner: ActiveWorkflowRunner,
+		private readonly nodeTypes: NodeTypes,
+		private readonly testWebhooks: TestWebhooks,
+		private readonly multiMainSetup: MultiMainSetup,
```

Comment:
This is definitely creating a bunch of circular dependencies. which is most definitely going to prevent the app from starting up.